api/v1alpha1/gatewayclass_types.go

```diff
@@ -73,6 +73,10 @@ type GatewayClassSpec struct {
 	// GatewayClass from any namespace. This field is intentionally not a
 	// pointer because the nil behavior (no namespaces) is undesirable here.
 	//
+	// When a Gateway attempts to use this class from a forbidden namespace, the
```

Comment:
suggest "from a namespace that is not allowed by this selector"

Reply:
Updated, thanks!